sdk/Pulumi/Provider/Provider.cs

```diff
@@ -863,7 +863,7 @@ public override async Task<Empty> Delete(Pulumirpc.DeleteRequest request, Server
         {
             try
             {
-                var domRequest = new DeleteRequest(request.Urn, request.Id, PropertyValue.Marshal(request.Properties), TimeSpan.FromSeconds(request.Timeout));
+                var domRequest = new DeleteRequest(request.Id, request.Urn, PropertyValue.Marshal(request.Properties), TimeSpan.FromSeconds(request.Timeout));
```

Comment:
We were passing the wrong thing to the `DeleteRequest` ctor. I noticed that we have a handful of `Request` classes that take `urn` and `id` in the ctor, but their order isn't consistent across the classes. Some take `urn` first, others take `id` first. Should we change them to have a consistent order?

Reply:
Yes URN first probably.

Reply:
Fixed up in https://github.com/pulumi/pulumi-dotnet/pull/83